internal/test/envtest/environment.go

```diff
@@ -36,8 +36,10 @@ import (
 )
 
 const (
-	capiPackage = "sigs.k8s.io/cluster-api"
-	capvPackage = "sigs.k8s.io/cluster-api-provider-vsphere"
+	capiPackage         = "sigs.k8s.io/cluster-api"
+	capdPackage         = "sigs.k8s.io/cluster-api/test"
+	capvPackage         = "sigs.k8s.io/cluster-api-provider-vsphere"
+	etcdProviderPackage = "github.com/mrajashree/etcdadm-controller"
```

Comment:
Should we point this to our build of this controller?

Reply:
we should
but our go.mod, and all our imports still point to this personal one
So, I would vote to make that change separately